tests/testproject/TestProject.WorkerA/Program.cs

```diff
@@ -6,5 +6,5 @@
 var builder = Host.CreateApplicationBuilder(args);
 builder.Services.AddHostedService<Worker>();
 
-var host = builder.Build();
+using var host = builder.Build();
```

Comment:
I assume there must be an analyzer somewhere that flags cases where you are not disposing IDisposable types. We should consider enabling that to prevent these test issues to happen again in the future. I'll log an issue to track this.